.github/workflows/tfupdate.yaml

```diff
@@ -10,10 +10,10 @@ jobs:
     steps:
 
       - name: "Checkout"
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
```

Comment:
Compare this change with (the expected/working) PR:
- https://github.com/MPV/renovate-pinGitHubActionDigestsToSemver-should-pinDigests-to-semver/pull/3